nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -223,16 +223,16 @@ public boolean dropNamespace(Namespace namespace) throws NamespaceNotEmptyExcept
 
   @Override
   public boolean setProperties(Namespace namespace, Map<String, String> properties) {
-    throw new UnsupportedOperationException(
-        "Cannot set properties for namespace '" + namespace +
-            "': setProperties is not supported by the NessieCatalog");
+    client.setProperties(namespace, properties);
```

Comment:
I would probably just have the client method return a boolean and then force the true in there rather than at this level in the Catalog. But that's just my opinion, up to you

Reply:
On this note we should probably raise another Issue to specify what the return of this method is expected to do.